ansible/roles/foundations/aws_permissions/files/policies.yml

```diff
@@ -58,15 +58,20 @@ Resources:
               - s3:GetObjectAcl
               - s3:GetObject
               - s3:ListBucketMultipartUploads
-              - s3:ListAllMyBuckets
               - s3:AbortMultipartUpload
-              - s3:ListBucket
-              - s3:HeadBucket
               - s3:ListObjects
               - s3:PutObjectAcl
               - s3:ListMultipartUploadParts
             Resource:
               - !Sub "arn:aws:s3:::${FilesBucket}/assets/*"
+          - Effect: Allow
+            Action:
+              - s3:ListBucket
+              - s3:HeadBucket
+              - s3:ListObjects
+            Resource:
+              - !Sub "arn:aws:s3:::${FilesBucket}"
```

Comment:
Does this not allow access to the top level bucket as well?

Reply:
If I'm reading [this](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-arn-format.html) correctly, this expression targets the bucket itself but not its content; just like `/assets` targets 'folder' within the bucket but, again, not its content. The `/assets/*`, on the other hand targets the content of 'assets' but not the folder itself. I think we're good :wink: